treeherder/perf/fixtures/performance_bug_templates.yaml

```diff
@@ -6,25 +6,13 @@
     default_component: Performance
     default_product: Testing
     text: |
-      Talos has detected a Firefox performance regression from push:
-
-      {{ revisionHref }}
-
-      As author of one of the patches included in that push, we need your help to address this regression.
+      Perfherder has detected a performance regression from push [{{ revisionId }}]({{ revisionHref }}). As author of one of the patches included in that push, we need your help to address this regression.
 
       {{ alertSummary }}
 
-      You can find links to graphs and comparison views for each of the above tests at: {{ alertHref }}
-
-      On the page above you can see an alert for each affected platform as well as a link to a graph showing the history of scores for this test. There is also a link to a treeherder page showing the Talos jobs in a pushlog format.
-
-      To learn more about the regressing test(s), please see: https://wiki.mozilla.org/TestEngineering/Performance/Talos
```

Comment:
This Talos description is pretty valuable. It's not mentioned anywhere in https://wiki.mozilla.org/TestEngineering/Performance/Handling_regression_bugs

We should update that page to include https://wiki.mozilla.org/TestEngineering/Performance/Talos also.

Reply:
I think @davehunt can tell you more about this.

Reply:
@ionutgoldan not sure what can we do here as https://wiki.mozilla.org/TestEngineering/Performance/Talos is outdated and very long. I wouldn't block this PR until that page gets up to date, could take months. What's your proposal?

Reply:
Sorry, I wanted to share the https://wiki.mozilla.org/TestEngineering/Performance/Talos/Tests link. I know this is the 1st thing I share to Firefox engineers for Talos-detected regressions.

Reply:
We can update [Talos regressions](https://wiki.mozilla.org/TestEngineering/Performance/Handling_regression_bugs#Talos_regressions) to contain this link. The idea is to have a single wiki page to reference the documentation pages of all frameworks.

Reply:
@ionutgoldan good enough? https://wiki.mozilla.org/TestEngineering/Performance/Handling_regression_bugs#Talos_regressions